scripts/push-docker.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
```

Comment:
Are you concerned that augur-core tag could be changed between building and pushing?

if they build then next day update package.json and then do a push